### PR TITLE
refactor test with POM

### PR DESCRIPTION
**Pull request by maria-martinezr**
image

### CODE REVIEW

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
-    cy.get(".cart_navigation span").click();
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+
+    productsListPage.addTShirtToCart();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin").click();
+    shoppingCartPage.proceedToCheckout();
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts:28]: this method, is the click over the proceed to checkout after the add to cart, it should be part of the productListPage object

File: cypress/e2e/buy-shirt.cy.ts [hunk 2]
```diff
@@ -1,29 +1,48 @@
...
+    shoppingCartPage.proceedToCheckoutSummary();
 
-    cy.get(".cart_navigation > .button > span").click();
+    loginPage.typeEmail("aperdomobo@gmail.com");
+    loginPage.typePassword("WorkshopProtractor");
+    loginPage.clickOnSignIn();
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: Thinking about the action that is behind these three methods, what we want is to login or signIn, so, we can simplify this, having just one method called login or signIn; email and password should be parameters of that method.

File: cypress/e2e/buy-shirt.cy.ts (hunk 1 above)
Comment (maria-martinezr) [reply cypress/e2e/buy-shirt.cy.ts:28]: Resolved!

File: cypress/e2e/buy-shirt.cy.ts (hunk 2 above)
Comment (maria-martinezr) [reply cypress/e2e/buy-shirt.cy.ts]: Resolved!

File: cypress/e2e/buy-shirt.cy.ts [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+
+    productsListPage.addTShirtToCart();
+    productsListPage.proceedToCheckout();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin").click();
+    shoppingCartPage.proceedToCheckoutSummary();
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: now we can rename this method as `proceedToCheckout`

Comment (maria-martinezr) [reply cypress/e2e/buy-shirt.cy.ts]: Resolved!

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**